CRM/Contact/Import/Parser/Contact.php

```diff
@@ -33,8 +33,6 @@ class CRM_Contact_Import_Parser_Contact extends CRM_Import_Parser {
   protected $_mapperRelatedContactDetails;
   protected $_relationships;
 
-  protected $_emailIndex;
-
   protected $_phoneIndex;
```

Comment:
Should `$_phoneIndex` also be removed here? Seems to be last reference in the repo.